backend/entities/organization_entity.py

```diff
@@ -43,16 +47,17 @@ class OrganizationEntity(EntityBase):
     heel_life: Mapped[str] = mapped_column(String)
     # Whether the organization can be joined by anyone or not
     public: Mapped[bool] = mapped_column(Boolean, nullable=False, default=True)
-
+    
+    # NOTE: This field establishes a one-to-many relationship between the organizations and events table.
+    events: Mapped[list['EventEntity']] = relationship(back_populates="organization", cascade="all,delete")
```

Comment:
Is `cascade="all,delete"` redundant in the `delete`?

Reply:
We found that we needed "all" for the cascading to work but will double check